backend_test.go

```diff
@@ -148,15 +150,45 @@ func (m *mockProvider) CreateApplication(ctx context.Context, parameters graphrb
 	}, nil
 }
 
+func (m *mockProvider) GetApplication(ctx context.Context, applicationObjectID string) (graphrbac.Application, error) {
+	return graphrbac.Application{
+		AppID: to.StringPtr("00000000-0000-0000-0000-000000000000"),
+	}, nil
+}
+
 func (m *mockProvider) DeleteApplication(ctx context.Context, applicationObjectID string) (autorest.Response, error) {
 	delete(m.applications, applicationObjectID)
 	return autorest.Response{}, nil
 }
 
+func (m *mockProvider) UpdateApplicationPasswordCredentials(ctx context.Context, applicationObjectID string, parameters graphrbac.PasswordCredentialsUpdateParameters) (result autorest.Response, err error) {
+	m.passwords = make(map[string]bool)
+	for _, v := range *parameters.Value {
+		m.passwords[*v.KeyID] = true
+	}
+
+	return autorest.Response{}, nil
+}
+
+func (m *mockProvider) ListApplicationPasswordCredentials(ctx context.Context, applicationObjectID string) (result graphrbac.PasswordCredentialListResult, err error) {
+	var creds []graphrbac.PasswordCredential
+	for keyID, _ := range m.passwords {
```

Comment:
FWIW, when you're ranging over a map and you only want the key but don't care about the value, you can just do:
```
for key := range someMap {
  // do stuff
}
```

Reply:
Oh yeah, thanks.